pages/vi/gitrepositories.md

```diff
@@ -12,57 +12,7 @@ On GitHub, our software code is organized in repositories, each of which represe
 
 You previously forked a repository from treehouses to work on, which means that that a version of the treehouses repository was available on your own GitHub account. You were then able to modify your fork, then send an update to the upstream repository in the form of a pull request. 
 
-## Summary of Steps
-Generally, this is what the structure of your workflow will look like: 
-
-![Image](images/upstream_diagram.png)
-
-Use the following commands in your command line, but refer back to the descriptions above if there are any errors or further questions about why you are using them:
-
-#### Clone your GitHub repository username.github.io
-1. Open your command prompt/terminal and find the correct directory
-2. Copy the HTTPS or SSH link from your repository on the GitHub site
-3. On the command line, type `git clone *paste your HTTPS or SSH link here*`
-
-#### Understand that there are three levels of a Github repository:
-- The upstream ([treehouses.github.io](https://github.com/treehouses/treehouses.github.io))
-- Your username.github.io on GitHub
-- Your username.github.io on your OS.
 
-These need to be synced and checked constantly.
-The **upstream repository** is the one we are contributing to.
-
-#### Configure the upstream repository to your fork
-1. `cd username.github.io`
-2. `git remote -v` see above to make sure you are pushing and fetching to your own repository on GitHub as the origin
-3. `git remote add upstream https://github.com/treehouses/treehouses.github.io.git`
-4. `git remote -v` origins should remain the same, but you should also be fetching and pushing to OLE as the upstream now
-
-#### Sync/Update Your Fork
-1. `git fetch upstream` -  to fetch branches from the upstream repository ([more info](https://git-scm.com/docs/git-fetch))
-2. `git checkout master` - to checkout the `master` branch ([more info](https://git-scm.com/docs/git-checkout))
-3. `git show-branch` - to see branches and the changes made in them ([more info](https://git-scm.com/docs/git-show-branch))
-4. `git merge upstream/master` - You repository should now be synced to upstream/master ([more info](https://git-scm.com/docs/git-merge))
-
-#### Make sure your repository is up to date
-1. `git diff` - for comparing different versions of the same file ([more info](https://git-scm.com/docs/git-diff))
-2. `git status` - to view the changes made in the branch, whether the branch is up-to-date with master ([more info](https://git-scm.com/docs/git-status))
-3. `git pull` - to sync the local repository with the remote repository ([more info](https://git-scm.com/docs/git-pull))
-4. `git push` - to push the updates that you made to the local repositories to the GitHub repositories ([more info](https://git-scm.com/docs/git-push))
```

Comment:
I think you should still include this section in the new update. It can help serve as a quick reference for anyone who may have forgotten about the steps and want to quickly check back.